### DISCUSSION TITLE
The problem with the "galaxy formation problem"

### DISCUSSION THREAD

**Discussion by mikehelland** [a-cosmology-group/acg#229] [Astrophysics and Physics]
From Anton Petrov two days ago on GS-z14:

https://www.youtube.com/watch?v=5B8-xUabZsw&t=522s

>It doesn't actually violate modern cosmological theories, as much as it basically presents a problem for modern theories in terms of galactic evolution. Or essentially this highlights, the models of galactic formations we currently use are most likely incorrect or incomplete.

Does that sound familiar at all? Here's what I've compiled this morning.  If you have any links/papers that I should add, by all means, leave a comment with them here.

----

### 2024
Galaxy formation models will need to address the existence of such large and luminous galaxies so early in cosmic history.
https://arxiv.org/abs/2405.18485

### 2023
Massive Optically Dark Galaxies Unveiled by JWST Challenge Galaxy Formation Models
https://arxiv.org/abs/2309.02492

### 2022
A semi-analytical perspective on the challenges of modern galaxy formation modeling – Doris Spoppacher
https://astro.uc.cl/en/charlas/a-semi-analytical-perspective-on-the-challenges-of-modern-galaxy-formation-modeling-doris-spoppacher/

### 2021
A challenge to models of star-formation truncation in massive galaxies
https://www.iac.es/en/outreach/news/challenge-models-star-formation-truncation-massive-galaxies

https://www.scientificamerican.com/article/giant-galaxies-from-the-universes-childhood-challenge-cosmic-origin-stories/

Speedy galaxy evolution - Mature features are detected in an early galaxy
https://www.science.org/doi/10.1126/science.abg2907

A massive stellar bulge in a regularly rotating galaxy 1.2 billion years after the Big Bang
https://www.science.org/doi/10.1126/science.abc1893
The authors found that the galaxy contains a massive stellar bulge and a regularly rotating disk, features that models predict take billions of years to form. These results indicate that galaxy evolution is a more rapid process than previously thought.

### 2020
The Rapid Buildup of Massive Early-type Galaxies: Supersolar Metallicity, High Velocity Dispersion, and Young Age for an Early-type Galaxy at z = 3.35
https://iopscience.iop.org/article/10.3847/1538-4357/abc7c4

Massive rotating disk galaxy challenges the traditional models of galaxy formation
https://www.mpia.de/5110677/20_05_nanowerk.pdf

What’s My Age Again? New Discoveries May Spark a Rethink of Galaxy Formation
https://www.zmescience.com/space/new-discoveries-may-spark-rethink-galaxy-formation/

Milky Way Look-Alike Challenges Theories on Early Galaxy Formation
https://www.courthousenews.com/milky-way-look-alike-challenges-theories-on-early-galaxy-formation/

ALMA Discovers Massive Rotating Disk in Early Universe
https://www.almaobservatory.org/en/press-releases/alma-discovers-massive-rotating-disk-in-early-universe/

A cold, massive, rotating disk galaxy 1.5 billion years after the Big Bang
https://www.nature.com/articles/s41586-020-2276-y

A dynamically cold disk galaxy in the early Universe
https://www.nature.com/articles/s41586-020-2572-6
https://skyandtelescope.org/astronomy-news/dead-ringer-milky-way-found-early-universe/

Web of the giant: Spectroscopic confirmation of a large-scale structure around the z = 6.31 quasar SDSS J1030+0524
https://www.aanda.org/articles/aa/full_html/2020/10/aa39045-20/aa39045-20.html

### 2019
Galaxy disc scaling relations: A tight linear galaxy–halo connection challenges abundance matching
https://www.aanda.org/articles/aa/full_html/2019/09/aa35982-19/aa35982-19.html

A dominant population of optically invisible massive galaxies in the early Universe
https://www.nature.com/articles/s41586-019-1452-4

### 2018
The strange case of quiescent and dusty
https://astrobites.org/2018/08/24/the-strange-case-of-quiescent-and-dusty/

A whirling plane of satellite galaxies around Centaurus A challenges cold dark matter cosmology
https://www.science.org/doi/10.1126/science.aao1858

The standard cosmological model of galaxy formation might be in trouble
https://citymonitor.ai/community/standard-cosmological-model-galaxy-formation-might-be-trouble-3661

New frontiers in galaxy and cluster formation in the early universe and challenges to theoretical models (special seminar)
https://www.phys.tsinghua.edu.cn/phyen/info/1056/1079.htm

### 2017
Theoretical Challenges in Galaxy Formation
https://arxiv.org/abs/1612.06891

### 2016
The galaxy counterpart of the high-metallicity and 16 kpc impact parameter DLA towards Q 0918+1636 – a challenge to galaxy formation models? 
https://academic.oup.com/mnras/article/464/2/2441/2404638?login=false

Forming Galaxies Without Bulges 
https://aasnova.org/2016/04/04/forming-galaxies-without-bulges/

### 2015
Outstanding challenges for galaxy formation models and simulations
https://centre-janssen.observatoiredeparis.psl.eu/-Outstanding-challenges-for-galaxy-formation-models-and-simulations-?lang=fr

Physical Models of Galaxy Formation in a Cosmological Framework 
https://ui.adsabs.harvard.edu/abs/2015ARA%26A..53...51S/abstract

The Birth of Monsters
https://www.eso.org/public/news/eso1545/
https://www.sciencedaily.com/releases/2015/11/151118070758.htm
To complicate things further, if massive galaxies are unexpectedly dustier in the early Universe than astronomers predict then even UltraVISTA wouldn't be able to detect them. If this is indeed the case, the currently-held picture of how galaxies formed in the early Universe may also require a complete overhaul.

### 2014
Nearby satellite galaxies challenge standard model of galaxy formation 
https://www.swinburne.edu.au/news/2014/06/nearby-satellite-galaxies-challenge-standard-model-of-galaxy-formation/

Dwarf galaxies don’t fit standard model
https://www.astronomy.com/science/dwarf-galaxies-dont-fit-standard-model/

STAR FORMATION AT 4 < z < 6 FROM THE SPITZER LARGE AREA SURVEY WITH HYPER-SUPRIME-CAM (SPLASH)
https://iopscience.iop.org/article/10.1088/2041-8205/791/2/L25
https://www.nasa.gov/jpl/spitzer/splash-project-dives-deep-for-galaxies
"The findings cast doubt on current models of galaxy formation, which struggle to explain how these remote and young galaxies grew so big so fast."

Three Ancient Halo Subgiants: Precise Parallaxes, Compositions, Ages, and Implications for Globular Clusters
https://arxiv.org/abs/1407.7591v1

### 2013
Dwarf galaxies orbiting Andromeda challenge galaxy formation theory 
https://pubs.aip.org/physicstoday/Online/17291/Dwarf-galaxies-orbiting-Andromeda-challenge-galaxy

### 2012
The current status of galaxy formation
https://iopscience.iop.org/article/10.1088/1674-4527/12/8/004/meta

GALAXY-SCALE STAR FORMATION ON THE RED SEQUENCE: THE CONTINUED GROWTH OF S0s AND THE QUIESCENCE OF ELLIPTICALS
Optical imaging reveals that all ETGs with galaxy-scale SF in our sample have old stellar disks (mostly S0 type). None is classified as a true elliptical. 

Galaxy Formation: Where Do We Stand?
https://arxiv.org/abs/1212.5641
"Finally, I review how observables of galaxies, and the observed galaxy formation process, compares with predictions from simulations of galaxy formation, finding significant discrepancies in the abundances of massive galaxies and the merger history. I conclude by examining prospects for the future using JWST, Euclid, SKA, and the ELTs in addressing outstanding issues"

### 2011
Atlas3d elliptals
https://www-astro.physics.ox.ac.uk/atlas3d/news.html

### 2010
A new measurement of the evolving near-infrared galaxy luminosity function out to z≃ 4: a continuing challenge to theoretical models of galaxy formation 
https://academic.oup.com/mnras/article/401/2/1166/1153869

Our Milky Way As A Pure-Disk Galaxy-A Challenge for Galaxy Formation
https://repositories.lib.utexas.edu/items/08c536ca-6342-4583-8a14-8b5d5518151e

BULGELESS GIANT GALAXIES CHALLENGE OUR PICTURE OF GALAXY FORMATION BY HIERARCHICAL CLUSTERING*,†
https://iopscience.iop.org/article/10.1088/0004-637X/723/1/54

### 2009
New Findings Challenge Galaxy Formation Ideas
https://www.universetoday.com/29585/new-findings-challenge-galaxy-formation-ideas/

New discovery challenges galaxy-formation theories
https://www.astronomy.com/science/new-discovery-challenges-galaxy-formation-theories/

Some Massive Galaxies May Be Relatively New: Discovery Challenges Galaxy Formation Theories
https://www.sciencedaily.com/releases/2009/04/090410123510.htm

### 2008
Constraining Galaxy Evolution With Bulge-Disk-Bar Decomposition
https://arxiv.org/abs/0802.3903
"We show that it is important to include the bar component, as this can significantly lower the bulge-to-total luminosity ratio (B/T), in many cases by a factor of two or more, thus effectively changing the Hubble type of a galaxy from early to late. "

### 2007
Very Massive Galaxies: A Challenge for Hierarchical Models?
https://inspirehep.net/literature/743954

### 2006
Breaking the hierarchy of galaxy formation 
https://academic.oup.com/mnras/article/370/2/645/967437

### 2005
The broken hierarchy of galaxy formation
https://inspirehep.net/literature/697716

### 2004
Astronomers see era of rapid galaxy formation
New findings pose a challenge for cold dark matter theory 
https://www.eurekalert.org/news-releases/726158

### 2003
Issues in E Galaxy Formation: Many Problems, Few Solutions
https://link.springer.com/chapter/10.1007/10857603_49

Simulations of Galaxy Formation in a Λ Cold Dark Matter Universe. II. The Fine Structure of Simulated Galactic Disks
https://iopscience.iop.org/article/10.1086/378316/meta
"Our results offer clues to understanding a number of observational trends that challenge the standard hierarchical disk assembly process"

### 2002
The Epochs of Early-Type Galaxy Formation
https://link.springer.com/chapter/10.1007/978-94-017-3311-3_83
"This finding is not compatible with the predictions from models of hierarchical galaxy formation."

### 2001
Toward Resolving the Mystery of Galaxy Formation
https://www.science.org/doi/10.1126/science.1065709

### 2000
https://iopscience.iop.org/article/10.1086/308225/meta
The Core Density of Dark Matter Halos: A Critical Challenge to the ΛCDM Paradigm?
"Reproducing the observed properties of disk galaxies thus appears to demand substantial revision to the currently most successful model of structure formation."

### 1999
Observing the epoch of galaxy formation
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC33559/

----

If anyone can help finding details from this talk:

2014
Quantitative predictions for galaxy formation
https://scienceandtechnology.jpl.nasa.gov/quantitative-predictions-galaxy-formation

That's be helpful because at least we can know what some of the actual predictions are supposed to be.

**Comment by ExpEarth** [comment]
Thanks for sharing Anton's video and also that list, Mike.  Anton seems like a straight-up guy compared to most.  I wonder when he will start to doubt the mantra.  For the moment, it seems that the mainstream can wiggle out of this by conjuring up new models of galaxy formation, to make them spring up so soon.

At the same time, I'm curious about how you and others here can explain these observations in a static model.  I think the relevant observations are these:

1.  The masses of galaxies at those redshifts (z = 9-14?) are the same or larger than the galaxies at low redshifts.
2. The masses of stars are also bigger than locally.
3. The amount of hydrogen gas is also greater than in local galaxies.
4. The number density of galaxies is relatively _constant_ at redshifts 9-14 (I read this somewhere but lost the reference)

It would be nice if we could attache more precise numbers to these points.  Adding it up, though, it seems like we can make the case that the baryonic mass density increases once we get to those high redshifts.  Does this sound right?

If that were true, this could lend support to my conjecture that the universe has a shell-like morphology, with most of its mass in a thick outer shell.  (That contrasts with the thin-shell model I discussed in #204.)  There is other evidence in favour of a very thick shell, which I will talk about pretty soon.  My basic point is, the excess hydrogen at those redshifts arises not because the universe was much younger then, but because hydrogen gas is mostly located at those redshifts _at all times_.  The stars would be brighter simply because there is more hydrogen there.

**Reply by mikehelland** [reply]
Howdy, Matt,

> 1. The masses of galaxies at those redshifts (z = 9-14?) are the same or larger than the galaxies at low redshifts.

Well, they aren't larger. They are in the range of what we see locally. See #221, and maybe post a comment there for follow up. 

> 2. The masses of stars are also bigger than locally.

I'm not sure about that. You might want to follow up in the mass-redshift thread with a reference on that too. Unless you're referring to Population III stars. They are hypothesized to be high-mass stars, but as of yet, they've never been observed.

> 3. The amount of hydrogen gas is also greater than in local galaxies.

Is that empirical or theoretical, based on metallicity increasing over time? A reference would be great for that too. I can say, that this article from 2024, seems to suggest we've got a lot more primordial gas than previously thought:

**Rethinking galactic origins: Heavy-element mapping challenges conventional theory**

_Takahiro Hayakawa from Nagoya University said, “The Galactic Fountain model describes the cycle of gas being blown out from the Galactic Plane by events like supernova explosions and then falling back, similar to how a fountain continuously reuses water. Researchers often cite the model for explaining IVCs because they believed these clouds to have metallicities similar to those of the stars that supposedly produced them.”_

https://www.techexplorist.com/rethinking-galactic-origins-heavy-element-mapping-challenges-conventional-theory/81989/

Paper:

https://academic.oup.com/mnras/article/529/1/1/7595798

> 4. The number density of galaxies is relatively constant at redshifts 9-14 (I read this somewhere but lost the reference)

The number density really depends on what you calculate as the volume between redshifts. Investigating that is what was behind the quasar numbers presented in #210.  

Maybe the reference you had in mind was this?

https://arxiv.org/abs/2405.13108

If not, let me know if you find it. Thanks!

**Reply by RedshiftDrift** [reply]
@ExpEarth 
You ask
> I'm curious about how you and others here can explain these observations in a static model. I think the relevant observations are these:
> 1) The masses of galaxies at those redshifts (z = 9-14?) are the same or larger than the galaxies at low redshifts.
> 2) The masses of stars are also bigger than locally.
> 3) The amount of hydrogen gas is also greater than in local galaxies.
> 4) The number density of galaxies is relatively constant at redshifts 9-14 (I read this somewhere but lost the reference)

These are not observations but the conclusions of analyses made with the $\Lambda$CDM model.  In a static model with tired light redshift:
1) distant galaxies have the same mass and size a local galaxies.  It's the "expanding space" assumption of $\Lambda$CDM that changes the metric at high redshift, making these galaxies ***appear*** massive.  @mikehelland is right, they are not.
2) the stars also have the same mass as those in local galaxies.  The problem here is that $\Lambda$CDM distorts the luminosity scale (because it believes light is attenuated as space expands), giving the ***illusion*** that bigger stars are needed to explain onservations.  This is not the case when observations are analysed with a static model.
3) The amount of hydrogen is the same at any distance.  Again, it is the analysis made with 'expanding space' that gives the ***illusion*** that the amount of hydrogen is higher in distant galaxies ($\Lambda$CDM calculates the same amount of hydrogen in a smaller space).  In a static model, the same amount of hydrogen is compatible with observations.
4) again, number density involves a calculation that uses the volume of space. @mikehelland has the right answer.  If data is analysed in the $\Lambda$CDM paradigm, space expands and galaxies are formed at a high rate at high redshifts.  Therefore $\Lambda$CDM calculates a roughly constant number density of galaxies independent of redshift at large z.  But the number density doesn't change in a static universe, this is also compatible with observations.

A simple static universe with Euclidean metric, tired-light redshift and *proper data analysis using the static universe paradigm* explains every observations reported by the Hubble and JWST telescopes, as well as the sky surveys (SDSS, DESI, etc.)

The galaxy models listed above were all adjusted to fit data analysed in the $\Lambda$CDM paradigm.  They need periodic revision as we observe more and more distant galaxies.  Anton Petrov only reports what's happening, without a clue that these constant revisions are necessary because the underlying $\Lambda$CDM model is wrong.

In a static model, the universe is old enough to account for any galaxy size at any distance.

**Reply by ExpEarth** [reply]
Thanks for your replies, @mikehelland and @RedshiftDrift.  I think that my post was not one of my better ones, and possibly one of my worst!  However, let me see if anything can be salvaged from it.

On point 1, the masses of distant galaxies, I think that I was being deluged with all the talk of "massive" galaxies found by JWST and not looking at the actual sizes.  That's a very helpful graph, Mike.  I do note that the galaxy masses actually seem to drop at the far edge.  That's interesting.  Thanks for setting me straight on this.

On point 2, the masses of stars, I also backtrack, for similar reasons.

On point 3, I have to go back and find the references.  I have read a few cases of galaxies having way more hydrogen gas than local galaxies.  Also, I seem to have a vague impression that clouds of hydrogen were more prevalent then.  I'll report back on this if I have anything.

On point 4, the number density of galaxies, @mikehelland that was not the reference, but thanks.  It was in a recent paper and I will try to find.  I confess I find it hard to imagine how they calculate this number with all their assumptions of expanding space, etc.  I was not thinking about discussion #210 , but glad you raised it.  To the extent that it is true that the luminosity peak of quasars moves with redshift, it could also conceivably be caused by more hydrogen being there.

@RedshiftDrift you mention it all makes sense if you use the static model paradigm.  Basically, I agree, but recent developments have made me see that we need to incorporate some elements of cosmic structure into it.  It also seems miraculous that they come up with these numbers indicating galaxy mass and number density are similar at high redshifts compared to low redshifts, given all their assumptions relating to volume, luminosities, etc.

**Reply by mikehelland** [reply]
>To the extent that it is true that the luminosity peak of quasars moves with redshift, it could also conceivably be caused by more hydrogen being there.

I don't think anyone here (me included) thinks that's true. It's a mis-calculated absolute magnitude due to model assumptions and selection effects. Thus they conclude the z > 1 quasars are deemed brighter than z < 1 quasars.

Take this quasar for example, the first one ever.

https://en.wikipedia.org/wiki/3C_273

>3C 273 is a [quasar](https://en.wikipedia.org/wiki/Quasar) located at the center of a giant [elliptical galaxy](https://en.wikipedia.org/wiki/Elliptical_galaxy) in the [constellation](https://en.wikipedia.org/wiki/Constellation) of [Virgo](https://en.wikipedia.org/wiki/Virgo_(constellation)). It was the first quasar ever to be identified and is the visually brightest quasar in the sky as seen from Earth, with an [apparent visual magnitude](https://en.wikipedia.org/wiki/Apparent_visual_magnitude) of 12.9.

It has as z = 0.158, and an absolute magnitude of −26.7.

Using my SDSS explorer tool:

https://mikehelland.github.io/hubbles-law/other/sdss.htm

Setting the absolute magnitude slider to -27 and checking +/- 0.5 should include it. But Vs[0], says there are zero quasars in that magnitude range with a z < 0.27 (the lowest redshift bin). We know there's at least one. Unless I'm doing something wrong, this quasar was simply not included in the dataset.

If we look at what SDSS DR16Q aims to do:

>Two distinct redshift ranges for quasars were identified as critical to the goals of eBOSS ([Dawson et al. 2016](https://ui.adsabs.harvard.edu/abs/2016AJ....151...44D/abstract)). Over a sky area of 7500 sq. deg., the eBOSS mission planned to observe more than 500,000 confirmed quasars in the range 0.8 < z < 2.2 and observe ~120,000 quasars for Lyα forest measurements in the range z > 2.1

https://www.sdss4.org/dr17/algorithms/qso_catalog/

Well, the quasar in question is z < 0.8. I think this quasar (and probably many more) are not in the dataset simply because they weren't interested in a comprehensive catalog, just the z ranges their missions targeted. 

I'm not saying this is negligent or fraudulent mind you. They didn't care about close by stuff, so it's not there. That's certainly their prerogative. Telescope time doesn't grow on trees, after all. 

That doesn't stop other people from using the dataset to draw conclusions about what is or isn't close by. That's also no malicious. Just.... misguided...

**Reply by ExpEarth** [reply]
@mikehelland 

`Well, the quasar in question is z < 0.8. I think this quasar (and probably many more) are not in the dataset simply because they weren't interested in a comprehensive catalog, just the z ranges their missions targeted.`

I think quasars have always been seen as 'cosmological', with redshifts generally z > 2.  The quasar in Virgo is surely an outlier and therefore an interesting object.

On point 3, the amount of hydrogen gas at high redshift, I just found this paper: Strong damped Lyman-α absorption in young star-forming galaxies at redshifts 9 to 11, by Heintz et al   https://doi.org/10.1126/science.adj0343  Here is the abstract:

`Primordial neutral atomic gas, mostly composed of hydrogen, is the raw material for star formation in galaxies. However, there are few direct constraints on the amount of neutral atomic hydrogen (H i) in galaxies at early cosmic times. We analyzed James Webb Space Telescope (JWST) near-infrared spectroscopy of distant galaxies, at redshifts ≳8. From a sample of 12 galaxies, we identified three that show strong damped Lyman-α absorption due to H i in their local surroundings. The galaxies are located at spectroscopic redshifts of 8.8, 10.2, and 11.4, corresponding to 400 to 600 million years after the Big Bang. They have H i column densities ≳1022 cm−2, which is an order of magnitude higher than expected for a fully neutral intergalactic medium, and constitute a gas-rich population of young star-forming galaxies.`

Does this not lend support to the idea that we have more hydrogen out there than locally?

I have a related question.  If this is true, that there are more damped Lyman-α galaxies out there at z > 8, then might these appear as smaller galaxies using other measurements?  Much of their light would be absorbed by the gas.  Perhaps this could account for the small galaxies at the far right of your graph in #221

**Reply by ExpEarth** [reply]
In #225 @mikehelland mentions this paper:

Do high redshift QSOs and GRBs corroborate JWST?
https://arxiv.org/abs/2405.19953

It has some remarkable conclusions, which seem to support the idea that the mass density of the universe increases with redshift.  Here is what they write:

`However, this may not be the full story. In [92] it was observed that Ωm increases with effective redshift in the QSO sample. Importantly, as we have seen in Fig. 3, QSOs recover the Planck value Ωm ∼ 0.3 at lower redshifts. In this paper we have revisited the observation with profile likelihoods and confirmed that the 68% confidence intervals are shifted to larger Ωm values as one increases the effective redshift of the sample. Moreover, profile likelihood peaks in Ωm > 1 parameter space are consistent with negative dark energy density. Since Ωm theoretically must be a constant in the ΛCDM model if matter is pressureless, which incidentally is an as- sumption that Bayesian cosmologists need never question, observationally Ωm need not be a constant.`

Check out Table 3, in particular.

**Reply by LifeIsStrange** [reply]
@ExpEarth Ωm is not the only thing to have been found to "increase with redshift", this is also the case of the  S8 parameter
https://arxiv.org/abs/2303.06928
https://arxiv.org/abs/2206.11447
https://www.youtube.com/watch?v=dSaflbDYCrk
https://www.prophy.science/article/176104667-Lambda-CDM-Tensions-Localising-Missing-Physics-through-Consistency-Checks/

"Results here and elsewhere suggest that the ΛCDM cosmological parameters are redshift dependent. Fitting parameters that evolve with redshift is a recognisable hallmark of **model breakdown.**"
You might interpret it as being weak evidence for the mass increase with redshift, or more likely, that the concordance LCDM model is broken and that it can no longer fit dynamical constraints despite trying (bruteforce) every permutations of its parameters.
Second paper I link shows that this LCDM hallucination can be reproduced with synthetic data.

edit note about the paper you linked, they say:
> One could more accurately determine the tension from the
MCMC chain, but it is clear that one needs to run a chain
long enough to encounter the Planck value or its 1σ upper
bound. What is important here is that we arrived at 3.9σ ten-
sion through a difference in the χ2 in section V, but it is clear
from our MCMC analysis that this number is a lower bound.
In principle, one could run a very long MCMC chain to see
whether the tension as ascertained by MCMC is closer to the
4.5σ we found with methodology based on equations (11) and
(12). 

So it is possible to increase the GRB tension from 3.9 to 4..5 sigma by doing a more bruteforce simulation it seems but they haven't tried this opportunity.

**Reply by LifeIsStrange** [reply]
DES latest release shows more evidence of increasing matter parameter with redshift, high redshift sn data is consistent with negative dark energy (no idea how negative dark energy would work regardless it is more evidence of lcdm being broken or of the signal for dark energy actually being a signal of increased reddening (dust attenuation) with distance))
Moreover the model is more consistent with a curved universe (like planck PR3 but not PR4?) and changing said parameter create more tensions.
> This is an irrefutable signature of model breakdown

https://arxiv.org/pdf/2406.06389
for dust evolution:
https://arxiv.org/abs/2406.06215
tests of other cosmologies (albeit no static universes/TL...)
https://arxiv.org/abs/2406.05048
inverse distance ladder
https://arxiv.org/abs/2406.05049

**Reply by ExpEarth** [reply]
They mentioned negative dark energy also in the paper mentioned above: Do high redshift QSOs and GRBs corroborate JWST?
https://arxiv.org/abs/2405.19953

I think the negative dark energy is introduced purely to eliminate the anomalously large values they are getting for Ωm at those high redshifts.

**Reply by LifeIsStrange** [reply]
Yes from my limited understanding negative dark energy is a natural mechanism to cope with the anomaly.
As the paper says, since the density matter parameter amalgamate in one parameter both the quantity of matter and the function of dark energy we cannot yet know wether the anomaly stems from a modified dark energy function or from a different matter estimate, especially the premise that matter is pressureless and they mention a way to test this.
This of course ignore dust obscuration and other systematics.
LCDM has finally broke with inconstant constants for all major probes (GRBs, QSOs, SNIa, BAOs), this is historic.

**Reply by mikehelland** [reply]
@ExpEarth Hello,

That shows that there were pristine clouds of gas back then, but I think there's plenty of it around us too. In the BBT, the universe began as ~75% hydrogen, and today it's ~74% H. In theory.

Take our own Milky Way. Check this out where they:

>interpret the finding of low-metallicity variations as the product of the falling of high-velocity clouds of pristine gas onto the Galactic disk. 

https://iopscience.iop.org/article/10.3847/1538-4357/ac6b38 

So pristine gas isn't particularly unusual. Also see #213 that includes extremely metal poor galaxies at z ~ 0. There has to be primordial gas every where for those to come about.

**Reply by mikehelland** [reply]
I could be wrong here, but I think "negative dark energy" isn't an entirely new kind of dark energy. It's the same ol' dark energy it has always been. It's equation of state has been $w=-1$ but now it might need to "wobble" a bit.

**Reply by LifeIsStrange** [reply]
Note that the paper says:
> a lack of late-time accelerated expansion in Table 2
[39], w(z = 0) = w0 ∼ 0 within 1σ, and a Universe younger
by 9% [39]. 
also

> curvature Ωk decreases with effective redshift disfavouring a flat Universe in higher redshift DES
SNe at > 3σ.

I don't understand why decreasing curvature with redshift would be evidence for a curved universe
or does effective redshift means gowing downwards to lower redshift?

**Reply by mikehelland** [reply]
It has to do with the FLRW models of the universe.

If $\Omega_\Lambda + \Omega_m + \Omega_r = 1$ it's called flat. 

If it's closed, it ends up like a sphere, and if it's open, it ends up like a Pringle. 

ETA: If the universe is flat $\Omega_k = 0$, and is less than 0 for a closed curved universe, and greater than zero for an open curved universe. Ωk = 1 - ΩΛ - Ωm - Ωr

**Reply by ExpEarth** [reply]
Thanks for all the info and refs!

Concerning the 'negative dark energy', I'm confused if that paper @LifeIsStrange mentions actually lumps matter and dark energy into a single parameter.  If we just use Mike's equation above, it's easier to understand.  If Ωm gets near to or greater than 1, then the universe will be super closed.  To try to get it flat, the ΩΛ term has to become really negative, which makes dark energy even stranger.

**Reply by ExpEarth** [reply]
Here is a paper which gives strong supporting evidence for the shell model.   It was in the s8 paper that @LifeIsStrange cited above. 

**Revealing Intrinsic Flat ΛCDM Biases with Standardizable Candles** https://arxiv.org/abs/2203.10558

They use a relationship between UV and Xray luminosities in QSOs to show that, assuming a flat LCDM, the universe transitions from a dark energy universe to an Einstein-deSitter universe at about z = 1.  What they mean is that the value Ωm moves from about 0.3, as in the LCDM model, to a value near 1 above _z_ = 1.  This also implies the Hubble constant starts to decrease at that redshift.  In the same paper they also say that SN data with Pantheon also supports this same trend.  (For some reason SN decrease in number above a certain redshift).  The SN data were used to make their Figure 2 below.  _zmin_ refers to all SN below that redshift.

Question: if the Hubble constant decreased in that way, could this alone explain the lower SN luminosities observed at high redshift that were used to justify dark energy in the first place?  The SN would be farther away than their redshift distances (using _H_ = 75) would indicate. 

![image](https://github.com/a-cosmology-group/acg/assets/136624758/c4d60d60-a4dd-41f1-9d78-fc5f719b4b90)

`FIG. 2. Variations of best-fit cosmological parameters ðH0;ΩmÞ
as low-redshift SN below zmin are removed. The central values of
H0 and Ωm favor lower and higher values at higher redshifts,
respectively. The error bars denote the variance in fitted H0 and
Ωm values at each zmin taken before performing a cubic
interpolation.`

**Reply by mikehelland** [reply]
>  if the Hubble constant decreased in that way, could this alone explain the lower SN luminosities observed at high redshift that were used to justify dark energy in the first place? 

The Hubble parameter has been decreasing. It will approach around 45 km/s/Mpc and mimic a pure dark energy universe. For that reason it's been called "asympotically de Sitter".

But lower luminosities? I was thinking that there were somewhat brighter. Unless I'm reading the first image of this this wrong:

https://physics.stackexchange.com/questions/300365/time-dilation-of-distant-cosmic-events-what-is-it

**Reply by ExpEarth** [reply]
Mike, let me ask that again in hopefully plainer English.  Let's put aside the idea of cosmic expansion and suppose a static universe, more or less.  In that case the observed SN luminosities at different redshifts reflect a permanent situation.  If the Hubble constant starts to diminish at high redshifts as in Fig. 2 above, would the observed luminosities be greater or lesser than if it was a constant Hubble constant (eg, 75) at all distances?

**Reply by mikehelland** [reply]
In that scenario, with an unknown/unstated mathematical model, I don't think it would matter. Luminosity is basically how much light the object puts out. It doesn't depend on how far away you are.

**Reply by ExpEarth** [reply]
Let me try a graphic depiction.  In the figure, we are at point 1, where _H_ is about 75.  Around the point z = .5, _H_ is diminishing and at point 2 it's about 68 or so.  Now if we looked at a star at point 2, and assumed _H_ was 75 also at that point, we would think that it was at the position 2'.  The observed luminosity from that star would seem dimmer than expected, as the star is farther away than what we think.  My scales are not exact.  My question is, could this explain the observation that led to the hypothesis of dark energy, that the distant SN were dimmer than expected?  And is this a good way to show it?

![image](https://github.com/a-cosmology-group/acg/assets/136624758/1f6a65aa-fbf2-43a4-88ce-4b4ea7532fe9)

**Reply by mikehelland** [reply]
@ExpEarth I'll follow up in a different thread

**Reply by ExpEarth** [reply]
@mikehelland Thanks for the feedback.  I will check that supernova result you mentioned.  However, I was really more interested in how this trend in _H_ would influence observations of SN in a _static universe_  .  I'm postulating that it would remove the need to have dark energy in the LCDM model, but would also explain dimmer SN seen at high redshift in a static model.

Note added:  Or do people already think the SN data are explained with a tired light model?

**Comment by RedshiftDrift** [comment]
@mikehelland This is a wonderful list!  Can I copy it and use it in the August *ACG Newsletter*?

**Reply by mikehelland** [reply]
Absolutely, it probably needs a little editing, but yeah.
